src/XIVLauncher.Core/Components/MainPage/MainPage.cs

```diff
@@ -698,6 +698,9 @@ public async Task<Process> StartGameAndAddon(Launcher.LoginResult loginResult, b
             throw new NotImplementedException();
         }
 
+        App.Settings.AdditionalArgs += $" UserPath={Program.CompatibilityTools.UnixToWinePath(App.Settings.GameConfigPath.FullName)}";
```

Comment:
This code is currently also being run on Windows, which is probably not intended

Reply:
Thanks, the logic has been moved into the Unix block a few lines up. Sorry about that!

Reply:
No worries, on windows it would still be expected for the setting to have an effect though I guess (sorry for nagging xD)

Reply:
Nah it's good, I assume the `UnixToWinePath` method would have caused an exception on Windows anyway and I just didn't consider it lol